src/org/usfirst/frc4904/robot/commands/HopperAgitate.java

```diff
@@ -0,0 +1,31 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.Hopper;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class HopperAgitate extends Command {
+	protected long lastSwitchTime;
+
+	public HopperAgitate() {
+		requires(RobotMap.Component.hopper);
+	}
+
+	@Override
+	protected void execute() {
+		if (System.currentTimeMillis() - lastSwitchTime > Hopper.AGITATE_DELAY) {
```

Comment:
Manually setting the hopper will cancel the hopper agitate command, so this is not necessary.